CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file. This projec
 - Updated PublishHelmChart.yml to support signing
 - Moved Helm Gitlab job from custom devops image to alpine/helm.
 - Added variable to easily be able to create signed Android apks without having to use git tags.
+- Added job to pubish apk/aars to Artifactory in the Android template
```

Comment:
Create a 3.x.x Changelog section,  add this line to the changelog, and note the breaking change of deployApk to deploy on this line.